rest/src/plugins/mosaicRestrictions/MosaicRestrictionsDb.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2016-present,
+ * Jaguar0625, gimre, BloodyRookie, Tech Bureau, Corp. All rights reserved.
+ *
+ * This file is part of Catapult.
+ *
+ * Catapult is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Catapult is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Catapult.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const catapult = require('catapult-sdk');
+const MongoDb = require('mongodb');
+
+const { mosaicRestriction } = catapult.model;
+const { Long } = MongoDb;
+
+class MosaicRestrictionsDb {
+	/**
+	 * Creates MosaicRestrictionsDb around CatapultDb.
+	 * @param {module:db/CatapultDb} db Catapult db instance.
+	 */
+	constructor(db) {
+		this.catapultDb = db;
+	}
+
+	/**
+	 * Retrieves mosaic restrictions of the given mosaic ids.
+	 * @param {array<object>} mosaicIds Given mosaic ids.
```

Comment:
can do this @param {Array.<module:catapult.utils/uint64~uint64>}, for the params further down as well